packages/vscode-css-in-js/src/extension.js

```diff
@@ -29,7 +29,7 @@ function activate (context) {
       // return if there's no editor or it's not a javascript file
       if (
         !editor ||
-        !/javascript|typescript/.test(editor.document.languageId)
+        !/javascript|typescript|javascriptreact/.test(editor.document.languageId)
```

Comment:
This is not really a required change.

![screen shot 2018-06-22 at 2 19 29 pm](https://user-images.githubusercontent.com/737065/41792582-593382f0-7627-11e8-96bc-61a939689c85.png)